charts/traction/templates/postgres_secret.yaml

```diff
@@ -9,4 +9,6 @@ metadata:
   namespace: {{ .Release.Namespace }}
 type: Opaque
 data:
-  postgresql-password: {{ include "getOrGeneratePass" (dict "Namespace" .Release.Namespace "Kind" "Secret" "Name" (include "postgresql.secret.name" .) "Key" "postgresql-password") }}
\ No newline at end of file
+  postgresql-password: {{ include "getOrGeneratePass" (dict "Namespace" .Release.Namespace "Kind" "Secret" "Name" (include "postgresql.secret.name" .) "Key" "postgresql-password") }}
+  database-user: cG9zdGdyZXM=
```

Comment:
Should we generate the user as well, rather than have it hard-coded? If we do use a hard-coded user, maybe a more explicit user (`traction` maybe?) could be better suited.